aries-site/src/pages/components/tabs.mdx

```diff
@@ -12,6 +12,9 @@ import {
   docs="https://v2.grommet.io/tabs?theme=hpe#props"
   figma="https://www.figma.com/file/Kp4dWyhUTnKIJ1Cg5CoL9o/HPE-Tabs-Component?node-id=112%3A510"
   grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/Tabs/Tabs.js"
+  template
+  height={{min: 'small'}}
+  showResponsiveControls={[]}
 >
   <TabsExample />
 </Example>
```

Comment:
I was unable to write this comment in the "Usage" section. Not to give you more work but since many teams are asking for more visuals and less content, I feel it might be good to add an example that visually shows which part is the container (the parent) and the tabs (children). In most of our new components, we're creating an anatomy of the different elements that live within that box or container. Let me know if you want to include that in this round and either I or Beza can help with that visual.

Reply:
One last thing, we should mention that the underline should be the length of the PageContent.

Reply:
@vavalos5 I've consolidated the guidance into one section and have mentioned that the bottom-border should extend the full width of the container. Let me know if this is clear:
https://deploy-preview-2537--keen-mayer-a86c8b.netlify.app/components/tabs?q=tabs#alignment

I think in this case the example is enough as is (you see the bottom underline stretching the available space).

Reply:
Touché! i missed that